### PR TITLE
updated hacktober fest info

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chaterjee <speak2adi@gmail.com>

**Description**
Updated hacktober fest date and info
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8ca7468eca0b44b6a0254513d25d41a305dda5d0 at: https://61560d7d68e8ca7a9a3fca3f--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ac5f54f4f01316773df300717f5e808c837dd375 at: https://6156119837904597ebfd1e67--layer5ng.netlify.app

File: src/collections/programs/hacktoberfest-2020/index.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Hacktoberfest 2020"
+title: "Hacktoberfest 2021"
```
Comment (leecalcote) [inline src/collections/programs/hacktoberfest-2020/index.mdx:2]: Do we mean to update the 2020 version or leave it in place?

File: src/collections/programs/hacktoberfest-2021/index.mdx [hunk 2]
```diff
@@ -0,0 +1,141 @@
+---
+title: "Hacktoberfest 2021"
+thumbnail: ./hacktoberfest2020.png
+permalink: "hacktoberfest/2020"
```
Comment (leecalcote) [inline src/collections/programs/hacktoberfest-2021/index.mdx:4]:
```suggestion
permalink: "hacktoberfest/2021"
```

File: src/collections/programs/hacktoberfest-2020/index.mdx (hunk 1 above)
Comment (warunicorn19) [reply src/collections/programs/hacktoberfest-2020/index.mdx:2]: It'll be better to just update it.

Comment (leecalcote) [reply src/collections/programs/hacktoberfest-2020/index.mdx:2]: We're sort of half way between 2020 and 2021 in this PR. Understanding the state of your sickly brain (completely understandable) at this point in time, I'll remind you to "scroll down" as you think this through.

Comment (leecalcote) [reply src/collections/programs/hacktoberfest-2020/index.mdx:2]: (I made a funny right there, in case you missed it)

Comment (warunicorn19) [reply src/collections/programs/hacktoberfest-2020/index.mdx:2]: actually, I didn't change the permalink because I kept the name of the folder the same 😅.

**Comment by l5io** [comment]
🚀 Preview for commit 7fd9d2152a00597c7c2c3b5a2e32d4ec1255891a at: https://6156164d76e8976a8cba183d--layer5ng.netlify.app

**Review by navendu-pottekkat — COMMENTED** [review]
https://github.com/search?q=org%3Alayer5io+label%3Ahacktoberfest&type=issues
This URL could be changed in beginner friendly issues to show issues from all orgs.

Update the table to match the one in [newcomers landing page](https://6156164d76e8976a8cba183d--layer5ng.netlify.app/community/newcomers).

Change the image to 2021 Hacktoberfest with this image.

Layer5 Hacktoberfest

**Comment by warunicorn19** [comment]
@navendu-pottekkat i've changed the help-wanted link to have issues of all the orgs.

**Comment by warunicorn19** [comment]
Couldn't run layer5.io locally so I've made the changes blindly, hopefully they are okay

**Comment by l5io** [comment]
🚀 Preview for commit 6dc21b45cc75724f223c2a9b54e7cefdba178adf at: https://6156c53e8725b59a30b10dcc--layer5ng.netlify.app

**Comment by navendu-pottekkat** [comment]
The image did not have transparent background. Fixed it.
Layer5 Hacktoberfest

**Comment by navendu-pottekkat** [comment]
Rest is good.

**Comment by l5io** [comment]
🚀 Preview for commit 7313b3bd4e3546bd6fbbabcee3545ad48f7bb9e8 at: https://6156c9e7c591accc2f0b6b7e--layer5ng.netlify.app

**Review by navendu-pottekkat — APPROVED** [review]
Looks good. Thank you @warunicorn19 🚀

**Pull request merged**